core/server/models/settings.js

```diff
@@ -125,10 +124,6 @@ Settings = ghostBookshelf.Model.extend({
             attrs.value = JSON.parse(attrs.value);
         }
 
-        if (attrs.value === 'null') {
-            attrs.value = null;
```

Comment:
@gargol Could you please confirm & double check in the morning? 

1. I am not aware of a report where people have "null" in their database, but it means NULL
2. I am not aware of a use case.

Maybe I forgot a use case/report.

Reply:
@kirrg001 Found one case where we end up with `"null"` value in the db. To reproduce it you need to roll back to `2.16.3` and do an import of previously public site (`is_private` setting flag has to be falsy). After import we end up having:
```
mysql> select `key`, `value` from `settings` where `value` = 'null';
+----------+-------+
| key      | value |
+----------+-------+
| password | null  |
+----------+-------+
```

Reply:
We should probably include a fix for this in our migrations as well?

Reply:
Thanks for testing btw :)

Reply:
Pretty sure it was `null` as returned from `grep -roP "password.{0,15}" ./import.json`:
```
password","value":null,
```